opencti-platform/opencti-graphql/src/database/stix-2-0-converter.ts

```diff
@@ -108,6 +108,7 @@ const buildStixObject = (instance: StoreObject): S.StixObject => {
     x_opencti_id: instance.id,
     spec_version: '2.0',
     x_opencti_type: instance.entity_type,
+    x_opencti_modified_at: convertToStixDate(instance.x_opencti_modified_at),
```

Comment:
why is it `modified_at` in 2.1 converter and `x_opencti_modified_at` in the 2.0 converter ?